app/main/routes.py

```diff
@@ -92,106 +96,232 @@ def accessibility():
+
+@bp.route("/browse/series/<uuid:_id>", methods=["GET"])
+@access_token_sign_in_required
+def browse_series(_id: uuid.UUID):
+    """
+    Render the browse series view page.
+
+    This function retrieves search results for a specific
+    record(s) based on the 'series_id' provided in the query parameters, and renders
+    the list of results on the 'browse-series.html' template.
+
+    Returns:
+        A rendered HTML page with series records.
+    """
+    form = SearchForm()
+    page = int(request.args.get("page", 1))
+    per_page = int(current_app.config["DEFAULT_PAGE_SIZE"])
+
+    series = Series.query.get(_id)
+    body = series.body
+    breadcrumb_values = {
+        0: {"transferring_body_id": body.BodyId},
+        1: {"transferring_body": body.Name},
+        2: {"series": series.Name},
+    }
+
+    filters = build_filters(request.args)
+    sorting_orders = build_sorting_orders(request.args)
+
+    query = build_browse_series_query(
+        series_id=_id,
+        filters=filters,
+        sorting_orders=sorting_orders,
+    )
+
+    browse_results = query.paginate(page=page, per_page=per_page)
+
+    total_records = db.session.query(
+        func.sum(query.subquery().c.records_held)
+    ).scalar()
+    if total_records:
+        num_records_found = total_records
+    else:
+        num_records_found = 0
+
+    return render_template(
+        "browse.html",
+        form=form,
+        current_page=page,
+        browse_type="series",
+        results=browse_results,
+        breadcrumb_values=breadcrumb_values,
+        filters=filters,
+        sorting_orders=sorting_orders,
+        num_records_found=num_records_found,
+        dict_args={k: v for k, v in request.args.items() if k not in "page"},
+    )
+
+
+@bp.route("/browse/consignment/<uuid:_id>", methods=["GET"])
+@access_token_sign_in_required
+def browse_consignment(_id: uuid.UUID):
+    """
+    Render the browse consignment view page.
+
+    This function retrieves search results for a specific
+    record(s) based on the 'consignment_id' provided in the query parameters, and renders
+    the list of results on the 'browse-consignment.html' template.
+
+    Returns:
+        A rendered HTML page with consignment records.
+    """
+    form = SearchForm()
+    page = int(request.args.get("page", 1))
+    per_page = int(current_app.config["DEFAULT_PAGE_SIZE"])
+
+    consignment = Consignment.query.get(_id)
+    body = consignment.series.body
+    series = consignment.series
+    breadcrumb_values = {
+        0: {"transferring_body_id": body.BodyId},
+        1: {"transferring_body": body.Name},
+        2: {"series_id": series.SeriesId},
+        3: {"series": series.Name},
+        4: {"consignment_reference": consignment.ConsignmentReference},
+    }
+
+    filters = build_browse_consignment_filters(request.args)
+    sorting_orders = build_sorting_orders(request.args)
+
+    # set default sort for consignment view
+    if len(sorting_orders) == 0:
```

Comment:
is there something unique about consingments sorting that means that we need to set the default sort explicitly out here as opposed to inside build_sorting_orders like the other route?

Reply:
as we are using generic sort from filter_sort_builder, if we want a default sorting on specific browse view, we have to define in the route of that view

Reply:
Okay, I noticed the other routes do not set a default sort order. Does that mean we don't care the default order for those other pages? I feel like we definitely need the default sort order of results to match the default sort option in the sort dropdown. Worth double checking this is correct and if anything needs fixing we can create a bug ticket for it

Reply:
ended up setting the default sort options in each view, so not creating ticket 👍